docs/AQL/master03-syntax.adoc

```diff
@@ -475,6 +491,72 @@ The `AVG()` function returns the average value of given _expression_ argument. T
+* *lookup*: Look-up a code; this will retrieve all the information concerning one particular code. Examples are retrieving the preferred form to display, synonyms, etc.
+* *translate*: Translate a code; this will translate (find an equivalent) code from one Value Set to another one based on a predefined mapping available in the external terminology service. Translation precision may not be limited to full equivalence and different kinds of mappings may be possible (wider meaning, equivalent, narrower meaning, etc).
+* *subsumes*: Subsumption testing; this will determine if a particular terminology concept is a subtype (is-a) of another one. For example, test in SNOMED-CT if `Myasthenia Gravis | 91637004` is a subtype of `autoimmune disease | 85828009` (i.e. test if 85828009 subsumes 91637004).
+
+The `driver` argument represents an identifier of the terminology server API. It provides the information to send requests related to the ValueSet and operation parameters to the the right terminology service. In addition, it allows the system to understand how to parse responses back from the terminology service.
+Some examples of mostly used drivers are:
+
+[width="100%",cols="2,4",options="header",]
+|===
+|Driver     |Description
+
+|http://hl7.org/fhir/4.0       |FHIR Terminology Service v4.0
+|http://hl7.org/fhir/3.0       |FHIR Terminology Service v3.0
+|http://hl7.org/fhir/1.0       |FHIR Terminology Service v1.0
+|ots.oceanhealthsystems.com    |Ocean Ontology Service
+|bts.better.care               |Better Terminology Service
+|dts4.apelon.com               |
+|dts5.apelon.com               |
+|===
+
+The `params_uri` argument consists of URI path and query sections in compliance with {rfc3986}[IETF RFC 3986]. Most common operations uses this to identify the value set on which to perform the operation. A value set may be a full code system such as the full set of codes in SNOMED-CT or LOINC. The value set URI may include the version and edition sections (e.g. Australian, US, etc.). When the release and version identifiers are not provided, it is up to the external terminology server to decide which default version will be used.
+Some examples of mostly used URI parameters are:
+
+[width="100%",cols="2,4",options="header",]
+|===
+|URI parameters     |Description
+
+|http://terminology.hl7.org/ValueSet/v3-FamilyMember       |
+|https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1010.2       |
+|http://snomed.info/sct/32506021000036107/version/20200331?fhir_vs=refset/1200161000168100       |implicit VS defined for Australian Refset for Vaccination Reason
+|http://snomed.info/sct     |
+|http://www.omim.org        |
+|http://fhir.de/CodeSystem/dimdi/icd-10-gm      |
+|===
+
+The `TERMINOLOGY` function can be used in two ways in the `WHERE` clause, as a right operand of `matches` operator:
```

Comment:
'as a right operand' -> 'as the operand'

Reply:
the 'right' was used here as as 'right-side', (opposite to 'left'). It is not needed?

Reply:
For me I think not, you could also put 'right-hand operand'. But 'right operand' sounds a little funny (this is English language stuff that has no rules, it's just cultural use;)

Reply:
ok